src/k8s/api/v1/types_test.go

```diff
@@ -3,7 +3,7 @@ package v1
 import (
 	"testing"
 
-	"github.com/canonical/k8s/pkg/utils/vals"
+	"github.com/canonical/k8s/api/v1/testutils"
```

Comment:
Revert this, and use `package v1_test` above (such that the test code can use `pkg/utils` without the package importing types.